default/stringtables/en.txt

```diff
@@ -4628,14 +4628,15 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
-SYSTEM_SUPERIORITY_TITLE
-System Superiority
-
-SYSTEM_SUPERIORITY_TEXT
-'''When an armed fleet set to aggressive mode arrives at a system currently unoccupied by hostile fleets, it establishes system control. Once system control has been established, [[metertype METER_SUPPLY]] propagation is interrupted for all other empires, industry generated on hostile planets within the system can only be used on the respective planet, and ground troop regeneration is stopped.
-Also, all starlane entries in the occupied system become guarded by the blockading force, so if hostile forces enter the system, they can exit only by the starlane entry by which they arrived. If more fleets from the same hostile empire arrive from different starlane entries, these entries will become available as an exit as well. If all hostile fleets are destroyed before new fleets arrive, the previously opened exits will become closed again.
-If fleets from two or more hostile empires arrive at an unoccupied system at the same time, neither such empire will establish system control over each other, although they will both establish control against all other empires.
-Space monsters can also establish system control. However, if an empire fleet arrives and stays at the system for one full turn, the monster's control of the system will be broken. If a space monster and an empire fleet arrive at the same time, the empire fleet will be able to establish system control despite the monsters presence.'''
+SYSTEM_BLOCKADE_TITLE
+System Blockade
+
+SYSTEM_BLOCKADE_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system currently unoccupied by hostile fleets, it establishes a blockade. If this happens, [[metertype METER_SUPPLY]] propagation is interrupted for all other empires, and industry generated on hostile planets within the system can only be used on the respective planet. If hostile planets come under attack, their meter growth (e. g. ground troops) will be suspended.
```

Comment:
all other empires? does it matter if they are at peace or allied? what constitutes a hostile fleet or planet? what is a respective planet? what does it mean to come under attack? That sentence seems like it needs an "Additionally, " at the start.